packages/create-plugin/templates/minimum/.eslintrc.js

```diff
@@ -1,5 +1,13 @@
-'use strict';
-
 module.exports = {
-  extends: ['@cybozu/eslint-config/presets/kintone-customize-es5']
+  extends: [
+    '@cybozu/eslint-config/globals/kintone.js',
+    '@cybozu/eslint-config/lib/base.js',
+    '@cybozu/eslint-config/lib/kintone.js',
+  ],
+  rules: {
+    'prettier/prettier': ['error', { singleQuote: true }],
+    'space-before-function-paren': 0,
+    'object-curly-spacing': 0,
+  },
+  plugins: ['eslint-plugin-prettier'],
```

Comment:
We can use `prettier.js` to solve the CI error.
https://github.com/kintone/js-sdk/actions/runs/7635159605/job/20800124180#step:8:136

```suggestion
  extends: [
    '@cybozu/eslint-config/globals/kintone.js',
    '@cybozu/eslint-config/lib/base.js',
    '@cybozu/eslint-config/lib/kintone.js',
    '@cybozu/eslint-config/lib/prettier.js',
  ],
  rules: {
    'prettier/prettier': ['error', { singleQuote: true }],
    'space-before-function-paren': 0,
    'object-curly-spacing': 0,
  },
```